modules/gapi/misc/python/shadow_gapi.hpp

```diff
@@ -5,11 +5,22 @@ namespace cv
 {
    GAPI_EXPORTS_W GCompileArgs compile_args(gapi::GKernelPackage pkg);
 
+   // NB: This classes doesn't exist in *.so
+   // HACK: Mark them as a class to force python wrapper generate code for this entities
    class GAPI_EXPORTS_W_SIMPLE GProtoArg { };
    class GAPI_EXPORTS_W_SIMPLE GProtoInputArgs { };
    class GAPI_EXPORTS_W_SIMPLE GProtoOutputArgs { };
-   class GAPI_EXPORTS_W_SIMPLE GRunArg {  };
+   class GAPI_EXPORTS_W_SIMPLE GRunArg  {  };
+   class GAPI_EXPORTS_W_SIMPLE GMetaArg {  };
 
    using GProtoInputArgs  = GIOProtoArgs<In_Tag>;
    using GProtoOutputArgs = GIOProtoArgs<Out_Tag>;
+
+   namespace gapi
+   {
+       namespace wip
+       {
+           class GAPI_EXPORTS_W_SIMPLE IStreamSourcePtr { };
+       }
+   }
```

Comment:
Can we enable
```cpp
GAPI_EXPORTS_W Ptr<IStreamSource> inline make_capture_src(const String& path)
```
if we define a shadow class without `std::enable_shared_from_this`?
```cpp
   namespace gapi
   {
       namespace wip
       {
           class GAPI_EXPORTS_W_SIMPLE IStreamSource { };
       }
   }
```

Reply:
IStreamSource is abstract

Reply:
Fixed